cmd/cmd.go

```diff
@@ -115,34 +118,34 @@ func DownloadDependency(dependencyName string) error {
 }
 
 // Runs go mod graph command and returns slice of the dependencies
-func GetDependenciesGraph() (map[string]bool, error) {
-	pwd, err := os.Getwd()
-	if err != nil {
-		return nil, err
-	}
-
-	projectDir, err := GetProjectRoot()
-	if err != nil {
-		return nil, err
+func GetDependenciesGraph(projectDir string) (map[string]bool, error) {
+	var err error
+	if projectDir == "" {
+		projectDir, err = GetProjectRoot()
+		if err != nil {
+			return nil, err
+		}
 	}
 
 	// Read and store the details of the go.mod and go.sum files,
 	// because they may change by the "go mod graph" command.
 	modFileContent, modFileStat, err := GetFileDetails(filepath.Join(projectDir, "go.mod"))
 	if err != nil {
-		return nil, err
+		log.Info("Dependencies were not collected because could not find go.mod in", projectDir)
```

Comment:
Dependencies were not collected because could not find go.mod in --> Dependencies were not collected for this build, since go.mod could not be found in